services/db/recovery.go

```diff
@@ -149,6 +150,24 @@ func alertOps(mode string) {
 	}
 }
 
+// sendRecoveryModeStat sends the recovery mode metric every 1 minute
+func sendRecoveryModeStat() {
+	for {
+		time.Sleep(1 * time.Minute)
+		recoveryModeStat := stats.NewStat("recovery.mode_normal", stats.GaugeType)
```

Comment:
It's not a big deal but do we need to initialize this object every time?

Reply:
Moved it out of the loop